README.md

```diff
@@ -36,6 +36,7 @@ The possible settings are:
 | `repositories:{full_name}:gitHubToken` | `string` | Token used to verify __outgoing__ requests to GitHub repository | ✓
 | `repositories:{full_name}:thirdPartyFolders` | `string` | Comma-separated list of folders in which to look for changed files in pull request to remind user to update License. | X
 | `repositories:{full_name}:claUrl` | `string` | The GitHub API URL to the CLA file in JSON form. If undefined it disables CLA checking. See [here](https://developer.github.com/v3/repos/contents/#get-contents) for how the URL should look. _Example:_ https://api.github.com/repos/AnalyticalGraphicsInc/cesium-concierge/contents/specs/data/config/CLA.json | X
+| `repositories:{full_name}:contributorsUrl` | `string` | The GitHub API URL to the `CONTRIBUTORS.md` file in JSON form. If `undefined`, checking for first time contributors will be disabled. | X
```

Comment:
Can we just make this the url relative to the root of the repository? It should be fairly easy to translate this to the GitHub API url.

Reply:
I figured I should have it as the API url to be consistent because that's what `claUrl` is as well. If we change this, would you make the `claUrl` also relative to the root of the repo?

Reply:
Ah, I didn't notice. `claUrl` is trickier because, at least in our case, it's in a different repo.

I think in this case, we should construct the API url and leave `claUrl` alone for now.